main.py

```diff
@@ -12,6 +12,42 @@
 
 # Import whatever libraries/modules you need
 
-import numpy as np
+import tensorflow as tf
+from tensorflow.keras.datasets import cifar10
+from tensorflow.keras.preprocessing.image import ImageDataGenerator
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.layers import Dense, Dropout, Activation, Flatten
+from tensorflow.keras.layers import Conv2D, MaxPooling2D
+cifar10 = tf.keras.datasets.cifar10 
 
-# Your working code here
+(x_train, y_train), (x_test, y_test) = cifar10.load_data()
+
+x_train = tf.keras.utils.normalize(x_train, axis=1)
+x_test = tf.keras.utils.normalize(x_test, axis=1)
+
+model = tf.keras.models.Sequential() 
+
+# First convolution layer
+model.add(Conv2D(256, (3, 3), input_shape=x_train.shape[1:]))
+model.add(Activation('relu'))
+model.add(MaxPooling2D(pool_size=(2,2)))
```

Comment:
Something for you to think about or do research: (write your answer in the comment)
What's Max Pooling 2D?
What does Max Pooling 2D do?

Reply:
Pooling is a method of downsampling the image by decreasing the number of pixels. For each 2x2 patch in the image, the max-pooling method selects the pixel with the highest value and replaces the other 3 pixels in the 2x2 patch with that value. 
Correct me if I'm wrong!